std/dynamic_library.zig

```diff
@@ -154,3 +163,27 @@ fn checkver(def_arg: *elf.Verdef, vsym_arg: i32, vername: []const u8, strings: [
     const aux = @intToPtr(*elf.Verdaux, @ptrToInt(def) + def.vd_aux);
     return mem.eql(u8, vername, cstr.toSliceConst(strings + aux.vda_name));
 }
+
+pub const WindowsDynLib = struct {
+    allocator: *mem.Allocator,
+    dll: windows.HMODULE,
+
+    pub fn open(allocator: *mem.Allocator, path: []const u8) !WindowsDynLib {
+        const wpath = try std.unicode.utf8ToUtf16LeWithNull(allocator, path);
```

Comment:
I think you want this pattern, as it avoids needing an allocator: https://github.com/ziglang/zig/blob/af3263d7a87c922cbcb047944ee94123c69cfac1/std/os/file.zig#L51-L52